openwisp_monitoring/monitoring/static/monitoring/css/chart.css

```
@@ -110,12 +110,12 @@
 #ow-chart-contents .circle > span {
   font-size: 0.15em;
   word-spacing: -0.75px;
-  color: rgba(0, 0, 0, 0.6) !important;
+  color: var(--ow-overlay-60) !important;
   font-size: 13px;
```

Biome (2.4.6)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

(lint/suspicious/noDuplicateProperties)